tests/php/LeftAndMainTest.php

```diff
@@ -80,8 +74,7 @@ protected function tearDown(): void
 
     public function testExtraCssAndJavascript()
     {
-        $admin = $this->objFromFixture(Member::class, 'admin');
-        $this->logInAs($admin);
+        $this->logInWithPermission('ADMIN');
```

Comment:
The admin fixture and the `logInWithPermission('ADMIN')` user have the same permissions, so no need for the fixture.